torch/csrc/jit/script/compiler.cpp

```diff
@@ -2455,9 +2460,17 @@ struct to_ir {
         auto kind = getNodeKind(tree->kind(), inputs.size());
         auto overload = getOperatorOverload(tree->kind(), inputs.size());
         auto named_values = getNamedValues(inputs, /*maybe_unpack=*/false);
+        auto class_arg_position = 0;
+
+        if (tree->kind() == TK_IN) {
```

Comment:
The class_arg_position is weird here. This _would_ be the appropriate time to flip the order and call the method. Previously it was invalid because the subexpressions were not yet emitted. But now they are (`getNamedValues` does it) and it would fit better in the API if it just called `std::swap` on named_values.